app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -174,8 +174,15 @@ def create
 
     # Pass along all params except the built in rails ones and excluded_params
     uri = encode_bbb_uri('create', server.url, server.secret, pass_through_params(excluded_params))
+    callback_data = nil
 
     begin
+      if params['meta_bn-recording-ready-url']
+        callback_attributes = { recording_ready_url: params['meta_bn-recording-ready-url'] }
+        callback_data = CallbackData.create!(meeting_id: meeting.id, callback_attributes: callback_attributes)
```

Comment:
At this point you already have the `meeting` object, so you should be able to use:
```ruby
callback_data = meeting.callback_data.create!(callback_attributes: callback_attributes)
```

Reply:
meeting is not an active record object so it wouldnt work